tests/testthat/line_breaks_and_other/base-pipe-line-breaks-out.R

```diff
@@ -88,6 +88,14 @@ blew(y = 2, x |>
   a |> c() + 1
 }
 
+{
+  (
+    # some comment
+
```

Comment:
This is expected. The new transformer shouldn't remove line breaks in between lines of code.